examples/environment.yml

```
@@ -20,7 +20,7 @@ dependencies:
  - networkx>=2.0
  - numba
  - numpy
- - pandas
+ - pandas >=0.24.1,<1.0
```

```suggestion
 - pandas >=0.24.1
```

(master should now be able to support Pandas >1.0, though this PR may need to be rebased)